<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QDialog" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::FocusPolicy::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(26, 25, 40);</string>
  </property>
  <widget class="QPushButton" name="Continue">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>570</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Unispace</family>
     <pointsize>18</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid;
color: rgb(255, 255, 255);
font: 700 18pt &quot;Unispace&quot;;
background-color: rgb(255, 0, 255);
Border-radius: 20px;</string>
   </property>
   <property name="text">
    <string>Continue</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>340</y>
     <width>250</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Unispace</family>
     <pointsize>28</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 28pt &quot;Unispace&quot;;
background-color: rgb(85, 0, 255);
color: rgb(85, 0, 255);

color:white;
</string>
   </property>
   <property name="text">
    <string>User Name:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_username">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>340</y>
     <width>371</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Unispace</family>
     <pointsize>32</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 0, 255);
font: 700 32pt &quot;Unispace&quot;;
color:white;
selection-background-color: rgb(255, 0, 255);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_password">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>400</y>
     <width>371</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Unispace</family>
     <pointsize>32</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 0, 255);
font: 700 32pt &quot;Unispace&quot;;
color:white;
selection-background-color: rgb(255, 0, 255);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>460</y>
     <width>371</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Unispace</family>
     <pointsize>32</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::WheelFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 0, 255);
font: 700 32pt &quot;Unispace&quot;;
color:white;
selection-background-color: rgb(255, 0, 255);</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::InputMethodHint::ImhPreferNumbers</set>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>13</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2025</year>
     <month>5</month>
     <day>13</day>
    </datetime>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>400</y>
     <width>250</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Unispace</family>
     <pointsize>28</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 28pt &quot;Unispace&quot;;
background-color: rgb(85, 0, 255);
color: rgb(0, 118, 0);
color:white;
</string>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>460</y>
     <width>250</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Unispace</family>
     <pointsize>28</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 28pt &quot;Unispace&quot;;
background-color: rgb(85, 0, 255);
color: rgb(0, 118, 0);
color:white;
</string>
   </property>
   <property name="text">
    <string>Date:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>10</y>
     <width>301</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/new/prefix1/aaa.PNG);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>-30</x>
     <y>640</y>
     <width>1371</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::HLine</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>20</number>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>-40</x>
     <y>140</y>
     <width>1351</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::HLine</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>20</number>
   </property>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>310</y>
     <width>701</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: transparent;
border: 5px;
border-color: rgb(255, 0, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>701</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border: 5px solid rgb(85, 0, 255); 
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>180</y>
     <width>381</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/new/prefix1/Screenshot 2025-05-03 yappin.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>frame_3</zorder>
  <zorder>Continue</zorder>
  <zorder>label</zorder>
  <zorder>lineEdit_username</zorder>
  <zorder>lineEdit_password</zorder>
  <zorder>dateEdit</zorder>
  <zorder>label_4</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>frame</zorder>
  <zorder>frame_2</zorder>
  <zorder>label_5</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
